meta-lmp-base/conf/distro/lmp.conf

```diff
@@ -58,3 +58,8 @@ OSTREE_WKS_BOOT_SIZE ?= "--size 200M --overhead-factor 1"
 # This is needed to move fontcache for all installed packages
 # from /var/cache/fontconfig to /usr/lib/fontconfig/cache
 FONTCONFIG_CACHE_DIR = "${libdir}/fontconfig/cache"
+
+# Exclude parsing mfgtool-files recipes
+BBMASK += " \
+    /meta-lmp/meta-lmp-bsp/dynamic-layers/stm-st-stm32mp/recipes-support/stm32-mfgtool-files/stm32-mfgtool-files_%.bb \
```

Comment:
Is FlashLayout_sd_stm32mp157c-dk2-optee.tsv.in only relevant with lmp-mfgtools? If so we can move the variable definition to meta-lmp-bsp/conf/machine/include/lmp-mfgtool-machine-custom.inc I believe, without requiring this bbmask.

Reply:
```
./meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:LMP_FLASHLAYOUT_TEMPLATE:stm32mp15-eval ?= "FlashLayout_emmc_stm32mp157c-ev1-optee.tsv.in"
./meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:LMP_FLASHLAYOUT_TEMPLATE:stm32mp15-disco ?= "FlashLayout_sd_stm32mp157c-dk2-optee.tsv.in"
./meta-lmp-bsp/conf/machine/include/lmp-mfgtool-machine-custom.inc:LMP_FLASHLAYOUT_TEMPLATE:stm32mp15-disco = "FlashLayout_stm32mp1-usb.tsv.in"
./meta-lmp-bsp/conf/machine/include/lmp-mfgtool-machine-custom.inc:LMP_FLASHLAYOUT_TEMPLATE:stm32mp15-eval = "FlashLayout_stm32mp1-usb.tsv.in"

```

Reply:
My suggestion is just to use two different variables, it should fix the issue